[G2M] Site theming - configure new jekyll theme

sallkall opened the pull request:
### Video Preview:
https://user-images.githubusercontent.com/33563070/142902915-dfd88d0a-d9d2-4c9b-8f58-ff51856f835d.mov

### Summary:
* New site theme based on the[ `jekyll-serif-theme`](https://github.com/zerostaticthemes/jekyll-serif-theme)
* 3 site sections - culture, development and hiring
* `.md` files reorganized and tagged by categories (current categories include `culture` and `development` )
* Customized typography for [headings](https://fonts.google.com/specimen/PT+Sans), [body text](https://fonts.google.com/specimen/PT+Serif) and [code](https://fonts.google.com/specimen/Fira+Code)
* Simple code block and inline code styling
* Home page includes:
    * introduction to `eq-engineering` + illustration
    * featured culture posts sourced from the `_cultures` folder
    * 3 highlighted eq-engineering philosophies 
 * Footer includes:
    * social links (github and medium)

woozyking reviewed (commented):
LICENSE should probably be placed elsewhere (it's not the content license but the theme's)

Also seo.yml and features.yml don't match our content I believe

sallkall commented:
> LICENSE should probably be placed elsewhere (it's not the content license but the theme's)
> 
> Also seo.yml and features.yml don't match our content I believe

@woozyking 

#### LICENSE
- Where would be a good place to put it? I can create a folder called `license` and move it in there so it's not in the root directory? 

#### seo.yml & features.yml
- In the sub-footer the link back to the original theme was using data in `seo.yml` but I removed it and made it inline in `sub-footer.html`
- removed `features.yml`

sallkall commented:
* Removed the link to theme from the footer and replaced with the creative commons license since we own the content
* Moved theme license to the `licenses` folder

woozyking commented:
> Where would be a good place to put it? I can create a folder called license and move it in there so it's not in the root directory?

I think we can either rename it to `THEME_LICENSE` or we can move that to `_layouts` (the closest to the _theme_ itself I suppose)

also the current root README.md links need to be relinked to the `_cultures` and `_developments` subpages (so it still works in the github view)

woozyking reviewed (approved):
looks great. perhaps `rebase -i` the last 3 commits and squash them into one

woozyking commented on _data/features.json:
```diff
@@ -0,0 +1,29 @@
+[
+    {
+      "title": "Create Efficiency",
+      "description": "We believe that efficiency promotes productivity.",
```
this doesn't make much sense. if the objective here is Productivity, perhaps something like `title: “Productivity"`, and `description: "We adapt conventions and automation to promote productivity"`

woozyking replied:
see https://eqworks.slack.com/archives/C02MNBJA33N/p1637619614033900 for detailed discussions
